ofrak_core/ofrak/cli/ofrak_cli.py

```diff
@@ -173,9 +173,9 @@ def run(self, ofrak_env: OFRAKEnvironment, args: Namespace):
             logging.warning("No disassembler backend specified, so no disassembly will be possible")
 
         for ofrak_pkg_name in ofrak_pkgs:
-            if os.path.exists(ofrak_pkg_name):
+            try:
```

Comment:
I don't think we should just try/except an import like this, it could add to the sys.path and generally have "weird" side effects. How about checking, in addition to the path existing, that the path is a file? The `_import_via_path` function also assumes the path is to a file, so this check also helps validate that.

If I understand the PR description, the problem is it tries to import a directory like `ofrak_ghidra` rather than the global Python package. Checking if the target is a file should fix that.

Reply:
Yeah I think so, it may be redundant but is fine.